defaults/kfddistribution-kfd-v1alpha2.yaml

```diff
@@ -135,6 +135,7 @@ data:
         retentionSize: 120GB
         storageSize: 150Gi
       alertmanager:
+        defaultRules: "enabled"
```

Comment:
Same as above

Reply:
I got an error when on "reducer" when try to put a boolean